crates/proposer-client/Cargo.toml

```diff
@@ -26,16 +26,28 @@ tonic.workspace = true
 tracing.workspace = true
 url.workspace = true
 
+# Ecosystem dependencies
+op-succinct-grpc = { git = "https://github.com/agglayer/op-succinct.git", tag = "v2.1.3-agglayer", package = "grpc" }
```

Comment:
I am a bit conflicted about having the dependency here vs. at the workspace level:

In favour of keeping it here:
* This should be the only crate which uses `op-succinct-grpc`. All other crates should issue proof requests through this crate. So the dependency should not be globally accessible.

In favour of putting it at the workspace level:
* All dependencies on `op-succinct` would be in one place, making it easier to keep the versions in sync.

I guess we will not get automated bumps from dependabot since we depend on a specific tag.

Thoughts?